### PR TITLE
formula(go): remove `-s -w` usage in std_go_args

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

needs https://github.com/Homebrew/brew/pull/15988

### CODE REVIEW

File: Formula/a/antibody.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ class Antibody < Formula
   depends_on "go@1.17" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s -w -X main.version=#{version}", "-trimpath", "-o", bin/"antibody"
+    system "go", "build", "-ldflags", "-X main.version=#{version}", "-trimpath", "-o", bin/"antibody"
```
Comment (bayandin) [inline Formula/a/antibody.rb:26]: We don't use `std_go_args` here, so need to move to a separate PR (with adding `std_go_args` itself)

File: Formula/g/gh.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ class Gh < Formula
   def install
     with_env(
       "GH_VERSION" => version.to_s,
-      "GO_LDFLAGS" => "-s -w -X main.updaterEnabled=cli/cli",
```
Comment (bayandin) [inline Formula/g/gh.rb:30]: There's no `std_go_args` here as well

File: Formula/k/kubernetes-service-catalog-client.rb [hunk 3]
```diff
@@ -25,7 +25,6 @@ def install
     ENV["NO_DOCKER"] = "1"
 
     ldflags = %W[
-      -s -w
```
Comment (bayandin) [inline Formula/k/kubernetes-service-catalog-client.rb:28]: And no `std_go_args` here

File: Formula/a/antibody.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/a/antibody.rb:26]: good catch, I will revert both changes

**Comment by chenrui333** [comment]
gonna close it for now, revisit after mass bottling done
